blocks/columns/columns.css

```diff
@@ -1,33 +1,169 @@
-.columns > div {
+.section.columns-container {
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.section.hero-container + .section.columns-container {
+  padding-top: 0;
+}
+
+.columns {
+  font-size: var(--body-font-size-s);
+}
+
+.columns h2 {
+  margin: 25px 0 0;
+  line-height: 1;
+  max-width: 500px;
+  text-transform: uppercase;
+  font-size: var(--heading-font-size-xxl);
+}
+
+.columns p > em:first-child:last-child {
+  font-size: var(--body-font-size-xs);
+}
+
+.columns-row {
+  padding: 60px 0;
   display: flex;
   flex-direction: column;
 }
 
-.columns img {
-  width: 100%;
+.columns-row img {
+  height: auto;
+  margin: auto;
+  max-width: 260px;
```

Comment:
I don't have an overview of whether columns are used anywhere else. But would it make sense to also limit this to `.home .columns-row` to not have this "hard-coded" size for other columns?